src/test/java/LoginTests.java

```diff
@@ -7,16 +7,19 @@
 
 import java.time.Duration;
 
-public class LoginTests {
```

Comment:
Please `import org.openqa.selenium.chrome.ChromeDriver;`

For example

```
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.chrome.ChromeOptions;
```

![image](https://user-images.githubusercontent.com/104761482/226201492-1c971612-449e-4a2e-bd95-5cece3702626.png)

![image](https://user-images.githubusercontent.com/104761482/226201602-e8a41880-3b47-4f3b-a336-517a574cbbb4.png)

![image](https://user-images.githubusercontent.com/104761482/226202158-510a417c-7c2e-4a2f-8ccf-04017c476674.png)

Reply:
It works now! Thank you!